tests/marqo_test.py

```
@@ -235,11 +229,13 @@ def create_cloud_index(self, index_name, settings_dict=None, **kwargs):
         """
```

We should throw an error if the cloud index doesn't yet exist

---

Make sure the  error message is helpful for new devs: they should know where to register their new index and index settings.